File: public/sdk/java/ariana_enterprise_java/src/main/java/br/com/arianamoveis/enterprise/Catalog.java
package br.com.arianamoveis.enterprise;
import java.util.*;
public class Catalog {
  private final ArianaEnterpriseClient client;
  Catalog(ArianaEnterpriseClient client){ this.client = client; }
  public Map<String,Object> push(List<Map<String,Object>> products) throws Exception {
    return client.request("POST", "/catalog/push", Json.of(Map.of("manufacturer","ariana_moveis","products",products)));
  }
}

File: public/sdk/php/ariana-enterprise-php/src/Catalog.php
<?php
namespace ArianaEnterprise;
class Catalog { public function __construct(private Client $client) {} public function push(array $products, string $manufacturer = 'ariana_moveis'): array { return $this->client->request('POST','/catalog/push',['manufacturer'=>$manufacturer,'products'=>$products]); } }
